tips-for-getting-your-next-developer-job.rst

```diff
@@ -0,0 +1,183 @@
+.. post:: February 13, 2019
+   :tags: advertising, sustainability, hiring
+   :author: David
+   :location: SAN
+
+
+Tips For Getting Your Next Developer Job
+========================================
+
+Over the last month, the Read the Docs team conducted 30-40 customer development interviews
+with hiring managers and recruiters
+at companies ranging from 5-person companies to the biggest names in tech.
+We wanted to learn more about hiring processes at various companies
+with the ultimate goal of building `a product to help companies find developers`_.
+
+Last time, we covered some :doc:`tips for hiring managers <lessons-from-hiring-manager-interviews>`
+based on what companies told us they were doing.
+This time, we put together **tips for candidates looking for their next job**
+based on insights we heard from hiring managers.
+
+Rather than covering how to crack developer interviews which is its own challenge
+or how to gain the necessary technical skills,
+these tips will focus on getting your foot in the door. Specifically:
+
+* Making your application stand out
+* Improving your application response rate
+* Getting to the interview process
+
+.. _a product to help companies find developers: https://readthedocs.org/sustainability/advertising/recruiting/?utm_medium=referral&utm_source=readthedocs-blog&utm_campaign=next-developer-job
+
+
+.. figure:: img/2019-facebook-job-opening.png
+   :alt: A Facebook job post on LinkedIn with almost 300 applications in 2 weeks
+
+   That's almost 300 applications in 2 weeks on just one of the job sites where this is posted.
+
+
+When applying, go deep not wide
+-------------------------------
+
+As a candidate, there is a temptation to play the numbers game and apply to as many jobs as possible.
+Many jobs sites make it easy to apply to many jobs quickly and this gives you that feeling that your job search is progressing.
+
+**Based on what hiring managers told us, playing the numbers game isn't the best approach.**
+While it is a bit of a job-seekers market, especially for more senior talent,
+companies are looking people who want to work for their company and who believe in their mission.
+
+.. pull-quote::
+
+    "I don't want candidates who drive by job sites and pepper every job they find"
+
+Specifically using some quotes from our interviews, 
```

Comment:
This line reads a bit odd, is it referring to the pull quote above? Probably a cleaner way to do that.

Reply:
It does feel a bit forced. I'll rework it.